templates/tftmpl/monitor_consul_kv.go

```diff
@@ -28,6 +28,10 @@ func (m ConsulKVMonitor) ServicesAppended() bool {
 	return false
 }
 
+func (m ConsulKVMonitor) SourceIncludesVariable() bool {
```

Comment:
Will add comment:

`SourceIncludesVariable returns whether or not to include the source variables in the template. For the case of a monitor, this always returns true and must be overridden to return based on other conditions.`